flipper-cache-store.gemspec

```diff
@@ -0,0 +1,24 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path('../lib/flipper/version', __FILE__)
+
+flipper_cache_store_files = lambda do |file|
+  file =~ /cache_store/
+end
+
+Gem::Specification.new do |gem|
+  gem.authors       = ['John Nunemaker']
+  gem.email         = ['nunemaker@gmail.com']
+  gem.summary       = 'ActiveSupport: Cache::Store adapter for Flipper'
```

Comment:
Any particular reason this is different from description other than the warning when building the gem from the gemspec? If not, I'd probably let them be identical.